qt-bugs@ issue : 48500
Qt Software task ID : 248643
bugs.kde.org number : none
applied: no
author: Laurent Montel <montel@kde.org>

Just a little patch to fix playground/base/plasma/applet/welcome compile with -pedantic

Index: src/3rdparty/webkit/WebKit/qt/Api/qwebsecurityorigin.h
===================================================================
--- src/3rdparty/webkit/WebKit/qt/Api/qwebsecurityorigin.h	(révision 939952)
+++ src/3rdparty/webkit/WebKit/qt/Api/qwebsecurityorigin.h	(copie de travail)
@@ -28,7 +28,7 @@
 namespace WebCore {
     class SecurityOrigin;
     class ChromeClientQt;
-};
+}
 
 class QWebSecurityOriginPrivate;
 class QWebDatabase;
Index: src/3rdparty/webkit/WebKit/qt/Api/qwebdatabase.h
===================================================================
--- src/3rdparty/webkit/WebKit/qt/Api/qwebdatabase.h	(révision 939952)
+++ src/3rdparty/webkit/WebKit/qt/Api/qwebdatabase.h	(copie de travail)
@@ -26,7 +26,7 @@
 
 namespace WebCore {
     class DatabaseDetails;
-};
+}
 
 class QWebDatabasePrivate;
 class QWebSecurityOrigin;
